lib/tic_tac_toe.rb

```diff
@@ -4,17 +4,31 @@
 
 class TicTacToe
   
-  def initialize(board=nil)
+  def initialize(board=nil, current_player=nil)
     board ||= [1, 2, 3, 4, 5, 6, 7, 8, 9]
     @board = board
     @message = Message.new
     @ui = UserInterface.new
     @game = GamePlay.new(@ui, @message)
+    current_player ||= 'X'
+    @current_player = current_player
+    @num_of_turns = 0
   end
 
   def play
     @ui.display_message(@message.lookup(:welcome))
     @ui.display_message(@message.lookup(:instructions))
     @ui.display_board(@board)
+    while @num_of_turns < 9 do
```

Comment:
As a heads up, this is sort of a place holder while I build out the game ending logic. I don't expect every game will have exactly 9 turns.